src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -139,12 +142,11 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                     string.Join(' ', environment.GetCommandLineArgs())
                 },
                 { "sourceRepoCommitId", sourceRepoCommitId },
+                { "platformName", this.PlatformName },
```

Comment:
Are we able to revert the changes in this file?

Reply:
In our discussion with Dan, we concluded to add the `platform` telemetry in the build. That is why I added it. But I will keep rest as it was.